doc/quickstart/configure.rst

```diff
@@ -699,11 +699,18 @@ Custom CMOR tables
 
 As mentioned in the previous section, the CMOR tables of projects that use
 ``cmor_strict: false`` will be extended with custom CMOR tables.
-By default, these are loaded from `esmvalcore/cmor/tables/custom
+For derived variables (the ones with ``derive: true``), the custom CMOR tables
+will always be considered.
+By default, these custom tables are loaded from `esmvalcore/cmor/tables/custom
 <https://github.com/ESMValGroup/ESMValCore/tree/main/esmvalcore/cmor/tables/custom>`_.
 However, by using the special project ``custom`` in the
 ``config-developer.yml`` file with the option ``cmor_path``, a custom location
-for these custom CMOR tables can be specified:
+for these custom CMOR tables can be specified.
+In this case, the default custom tables are extended with those entries from
+the custom location (in case of duplication, the custom location tables take
+precedence).
```

Comment:
is that a good idea? I'd argue CMOR stock should be the first port of entry

Reply:
I think it is; it gives the user the option change an existing custom variable. For example, this might be necessary if a native model uses a variable name that we also have in our custom CMOR tables, but e.g. with different dimensions.

Reply:
indeed - I can see the benefits, but I can also see the negatives (stuff that can not be reproduced anymore bc the user has left with their tables). At any rate, it's philosophical at this point, the feature's in and this PR improves it :+1: